__tests__/helpers/daily-data-providers/combined-resting-heart-rate.test.ts

```diff
@@ -92,37 +208,65 @@ export function buildOuraDataPoint(value: string, startDate: string, propertyVal
         observationDateOffset: "",
         insertedDate: "",
         modifiedDate: "",
-        properties: { type: propertyType, day: startDate }
+        properties: properties
     } as DeviceDataV2Point;
-    ddp.properties![propertyValueName] = value;
-    return ddp;
 }
 
-export function buildFitbitDataPoint(value: string, startDate: string, ddpType: string): DeviceDataPoint {
-    let ddp: DeviceDataPoint = {
+function buildBaseDeviceDataPoint(
+    value: string,
+    startDate: string,
+    namespace: DeviceDataNamespace,
+    type: string,
+    properties: Record<string, any> = {}
+): DeviceDataPoint {
+    return {
         id: "",
-        namespace: "Fitbit",
+        namespace: namespace,
         insertedDate: "",
         modifiedDate: "",
         startDate: startDate,
-        type: ddpType,
+        type: type,
         value: value,
-        properties: {},
+        properties: properties
     };
-    return ddp;
 }
 
-export function buildGarminDataPoint(value: string, startDate: string, ddpType: string, propertyValueName: string): DeviceDataPoint {
-    let ddp: DeviceDataPoint = {
-        id: "",
-        namespace: "Garmin",
-        insertedDate: "",
-        modifiedDate: "",
-        startDate: startDate,
-        type: ddpType,
-        value: value,
-        properties: {},
-    };
-    ddp.properties![propertyValueName] = value;
-    return ddp;
-}
\ No newline at end of file
+function buildOuraDataPoint(
```

Comment:
There is some similarity between this and it's combined-steps.test.ts counterpart. Can these be combined into one, more general helper for use across this and other tests